ksqldb-engine/src/test/java/io/confluent/ksql/function/UdtfLoaderTest.java

```diff
@@ -97,13 +97,13 @@ public void shouldLoadParameterizedMapParams() {
 
```

Comment:
Not your code but I notice that `shouldLoadParameterizedListParams()` (the test above this one in this file) doesn't test what it says it's testing. Wanna fix it while you're in here?

Reply:
done